public/js/entry.js

```diff
@@ -1,5 +1,5 @@
 angular.module('main')
-  .controller('EntryCntrl', ['$scope', '$firebase', 'fbTop', function($scope, $firebase, fbTop){
```

Comment:
Having an n makes the file larger, therefore causes the project to load slower? :smile: But, seriously, there is a "ctrl" key on keyboards, no?